scripts/build/libc/glibc.sh

```diff
@@ -247,6 +247,10 @@ do_libc_backend_once() {
     [ -n "${CT_TOOLCHAIN_BUGURL}" ] && extra_config+=("--with-bugurl=${CT_TOOLCHAIN_BUGURL}")
 
     touch config.cache
+
+    # Hide host C++ binary from configure
+    echo "ac_cv_prog_ac_ct_CXX=" >>config.cache
```

Comment:
So, just to be clear: the resulting *final* toolchain still supports C++ and you tested it, right?